testing/src/test/java/io/stargate/it/http/HealthCheckerTest.java

```diff
@@ -85,4 +86,10 @@ public void healthCheck() throws IOException {
         .containsEntry("healthy", true)
         .containsEntry("message", "Ready to process requests");
   }
+
+  @Test
+  public void metrics() throws IOException {
+    // we care only for 200
+    RestUtils.get("", String.format("%s:8084/metrics", host), HttpStatus.SC_OK);
+  }
```

Comment:
Sure I'll add some meaningful stuff here, but won't overdo it.. Maybe confirm that modules expose at least one metric or smth like that.. We did not had those tests before, so I thought it was not a major issue..

Reply:
Not a major issue, but it should help to ensure we did not break metric names :) Ideally the same test should pass before and after this PR, so it might be worth making a separate PR for it merge it to `master` then rebase this PR and retest?

Reply:
Yea, but we are just doing an approximate check here.. If there are some metrics missing we would still not being able to find it with this test.. imo, I would not do this, effectively there were no chnages on what's collected, only how it's exposed.